doc/api/esm.md

```diff
@@ -579,6 +579,65 @@ node b.mjs # works
 
 <i id="esm_experimental_loaders"></i>
 
+## HTTPS and HTTP imports
+
+> Stability: 1 - Experimental
+
+Importing network based modules using `https:` and `http:` is supported under
+the `--experimental-https-modules` flag. This allows web browser-like imports to
+work in Node.js with a few differences due to application stability and security
+concerns that are different when running in a privileged environment instead of
+a browser sandbox.
+
+### HTTP is limited to loopback addresses
+
+`http:` is vulnerable to man in the middle attacks and is not allowed to be used
+for addresses outside of the IPv4 address 127.0.0.1 and the IPv6 address ::1 .
+Support for `http:` is intended to be used for local development.
```

Comment:
We should likely stress that only http1 is supported right now.

Reply:
should we add http2 here?

Reply:
Eventually... http3 also, but we need to work on the transparent discovery and protocol selection work to be implemented (it's on my list but a long ways out)